Modules/_testinternalcapi.c

```diff
@@ -2034,6 +2034,24 @@ gh_119213_getargs_impl(PyObject *module, PyObject *spam)
     return Py_NewRef(spam);
 }
 
+/*[clinic input]
+is_version_overflowed
```

Comment:
"version" is a generic term. I suggest to be more explicit, such as: `is_dict_version_overflowed`.

Reply:
Done